src/Storages/RabbitMQ/StorageRabbitMQ.cpp

```diff
@@ -708,7 +773,7 @@ void StorageRabbitMQ::streamingToViewsFunc()
 
     /// Wait for attached views
     if (!stream_cancelled)
-        streaming_task->schedule();
+        streaming_task->scheduleAfter(RESCHEDULE_MS);
```

Comment:
I think this was the main reason for CPU consumption without views?

Reply:
Yes